content_scripts/vimium_frontend.coffee

```diff
@@ -178,6 +183,7 @@ initializePreDomReady = ->
     currentKeyQueue: (request) ->
       keyQueue = request.keyQueue
       handlerStack.bubbleEvent "registerKeyQueue", { keyQueue: keyQueue }
+    frameFocused: -> # A frame has received the focus.  We don't care, here. The Vomnibar/UI-component cares.
```

Comment:
No.  But the `onMessage` handler (just below) doesn't check to see if a handler is defined.  So I added one here.

(It's arguably better not to check.  That way, we get errors when we make a mistake.)